primitives/src/transaction.rs

```diff
@@ -529,12 +529,34 @@ impl Version {
 
     /// The third Bitcoin transaction version (post-BIP-431).
     pub const THREE: Self = Self(3);
+
+    /// Returns the inner u32 value of the Version
+    #[inline]
+    pub const fn to_u32(self) -> u32 { self.0 }
+
+    /// Constructs a potentially non-standard transaction version.
+    ///
+    /// This can accept both standard and non-standard versions.
+    #[inline]
+    pub fn maybe_non_standard(version: u32) -> Version { Self(version) }
```

Comment:
nit: Since this is a constructor it'd be nicer IMO if it was above the getter. That is how I've tried to lay things out in other places in the codebase.